website/_dogfooding/_docs tests/test-unlisted.md

```diff
@@ -0,0 +1,7 @@
+---
+unlisted: true
+---
```

Comment:
Worth creating more dogfood cases. You can create a whole dogfood sidebar category with a few unlisted cases docs inside